WordPress/src/main/java/org/wordpress/android/push/GCMRegistrationIntentService.java

```diff
@@ -42,13 +40,7 @@ public static void enqueueWork(Context context, Intent work) {
     @Override
     protected void onHandleWork(@NonNull Intent intent) {
```

Comment:
On a clean installation I saw this method called at the startup of the app (when the user was not logged in to WP), the token was correctly retrieved from Google, but `sendRegistrationToken` did nothing since there was no wpcom access token in place.

After that this method was not called again. I had to restart the app by killing it manually.  This is related to my previous ticket #7873, but a bit worse since moving the app to FG doesn't trigger it.

Reply:
I think it's related to that other ticket, but not specifically related to this PR. Maybe we can merge and treat it separately? I'm opening a new PR targeting #7873 for that 👍